contrib/dockerd-rootless.sh

```diff
@@ -79,7 +85,7 @@ if [ -z $_DOCKERD_ROOTLESS_CHILD ]; then
 		--net=$net --mtu=$mtu \
 		--slirp4netns-sandbox=$DOCKERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SANDBOX \
 		--slirp4netns-seccomp=$DOCKERD_ROOTLESS_ROOTLESSKIT_SLIRP4NETNS_SECCOMP \
-		--disable-host-loopback --port-driver=builtin \
+		--disable-host-loopback --port-driver=$DOCKERD_ROOTLESS_ROOTLESSKIT_PORT_DRIVER \
```

Comment:
With more and more env-vars being added for this, would it make sense to;

- have a configuration-file for `rootlesskit`
- make `rootlesskit` _itself_ support env-vars for important options (so that we can drop `--port-driver` here entirely, and it would default to `builtin`, but optionally override with (e.g.) `ROOTLESSKIT_PORT_DRIVER` ?

Reply:
I think I'm not going to add more flags, but if we want to add more in future, we can consider supporting some configuration file.

Reply:
Yeah, was more thinking that if the binary would support its own env-vars, that we wouldn't have to invent them here, and users could (almost) just set them in their `~/.profile` and run `exec rootlesskit ..` without the shell script. Looks like some other things still need to be set up, so perhaps not possible